metrics/web/web/icons/avatar_hover.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
```

Comment:
should we rename this file to avatar_active.svg? As it's used when menu is active - not when the user hovers.